driver/kubernetes/driver.go

```diff
@@ -229,11 +229,18 @@ func (d *Driver) Factory() driver.Factory {
 }
 
 func (d *Driver) Features() map[driver.Feature]bool {
+	var historyAPI bool
+	ctx := context.Background()
+	c, err := d.Client(ctx)
+	if err == nil {
+		historyAPI = driver.HistoryAPISupported(ctx, c)
+		c.Close()
+	}
 	return map[driver.Feature]bool{
 		driver.OCIExporter:    true,
 		driver.DockerExporter: d.DockerAPI != nil,
-
-		driver.CacheExport:   true,
-		driver.MultiPlatform: true, // Untested (needs multiple Driver instances)
+		driver.CacheExport:    true,
+		driver.MultiPlatform:  true, // Untested (needs multiple Driver instances)
```

Comment:
Unrelated, but I think this comment is out-of-date now?

I think this is tested now, at least we have docs for this scenario?

Reply:
@AkihiroSuda @morlay